databricks/koalas/plot/plotly.py

```diff
@@ -92,3 +103,70 @@ def plot_histogram(data, **kwargs):
     fig["layout"]["xaxis"]["title"] = "value"
     fig["layout"]["yaxis"]["title"] = "count"
     return fig
+
+
+def plot_box(data: Union["ks.DataFrame", "ks.Series"], **kwargs):
+    import plotly.graph_objs as go
+
+    if isinstance(data, pd.DataFrame):
+        raise RuntimeError(
+            "plotly does not support a box plot with Koalas DataFrame. Use Series instead."
+        )
+
+    # 'whis' isn't actually an argument in plotly (but in matplotlib). But seems like
+    # plotly doesn't expose the reach of the whiskers to the beyond the first and
+    # third quartiles (?). Looks they use default 1.5.
+    whis = kwargs.pop("whis", 1.5)
+    # 'precision' is Koalas specific to control precision for approx_percentile
+    precision = kwargs.pop("precision", 0.01)
```

Comment:
Cool! May I ask where the default value(0.01) from ??

Reply:
It's a default precision for `approx_percentile` expression in PySpark, as written in the comment above.

Reply:
Got it! Thanks